provisioning/device/test/_polling_state_machine_test.js

```diff
@@ -14,6 +15,10 @@ var fakeRequest = {
   provisioningHost: '__FAKE_HOST__',
   idScope: '__FAKE_SCOPE__'
 }
+var fakeError = new Error(fakeErrorText);
+
+assert.isNotAnError = function(err) { assert.isNotOk(err, 'expected success, got an error.'); };
+assert.isAnError = function(err) { assert.isOk(err, 'expected an error.  got success'); };
```

Comment:
uhhh modifying a variable in the global scope... not a generally good practice

how about using assert.instanceOf(err, Error)?